### DISCUSSION TITLE
Saveback und Round up

### DISCUSSION THREAD

**Discussion by matawed** [pytr-org/pytr#63] [Ideas]
Does anybody know how one can retrieve documents related to 

- Saveback https://support.traderepublic.com/en-ie/1215-What-is-Saveback

and

- Round up https://support.traderepublic.com/en-ie/1216-What-is-Round-up

I mean how can we enable pytr to also retrieve that information?

**Comment by Katzmann1983** [comment]
It is available in the new version 0.2.0 :dancers:
